store/tikv/tests/util_test.go

```diff
@@ -114,3 +117,12 @@ func valueBytes(n int) []byte {
 func s08d(prefix string, n int) string {
 	return fmt.Sprintf("%s%08d", prefix, n)
 }
+
+func toTiDBTxn(txn *tikv.TxnProbe) kv.Transaction {
+	return txndriver.NewTiKVTxn(txn.KVTxn)
+}
```

Comment:
I'm confused the function name is `toTiDBTxn`, but it calls `NewTiKVTxn`.

Reply:
Here we need a `tidb/kv.Transaction`, however, I have no idea about the function name